### ISSUE TITLE
[Question] how to sold Jasper ASR Loss: inf or Gradient overflow issue?

### ISSUE THREAD

**Issue by caohongnga** [NVIDIA/NeMo#1513]
Hello,
I train Jasper ASR but the loss is inf and Gradient overflow. How should I solve it.
I already refer to the issue #637 and see that it is already fixed as suggested for the current version but my model still has the same issue.
Thank you

**Comment by okuchaiev** [comment]
try very small learning rate first. And / or learning rate policy with a longer warmup period. Then see if you can increase them.
